ch.threema.threema-web-desktop.yml

```diff
@@ -0,0 +1,79 @@
+app-id: ch.threema.threema-web-desktop
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-download
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    # Must be set according to upstream: https://github.com/threema-ch/threema-web-electron/pull/4#issuecomment-995968724
+    DEV_ENV: development
+    threema_web_version: threema-web-2.3.19
```

Comment:
maybe you can add `DESTCPU` defined depending on the architecture?

Reply:
Hm, but how do I do that?  That is, how can I set values/execute code depending on the architecture?  I know that `flatpak-builder` is passed the `--arch` flag and it seems that indeed Electron seems to correctly "autodetect" this and builds for aarch64/amd64, but I have no idea how this detection works in order to actually retrieve the correct file.

Reply:
something like that (adjust with your indent spacing):
```yaml
arch:
  x86_64:
    env:
      DESTCPU: x86_64
  aarch64:
    env:
      DESTCPU: aarch64
```

This should set `DESTCPU` to the right value.

Reply:
just to clarify the `arch` above is at the same level as `env` and `append-path` in the `build-options`

Reply:
I see, thanks!